airbyte-integrations/connectors/source-zuora/source_zuora/zuora_auth.py

```diff
@@ -0,0 +1,63 @@
+#
+# MIT License
+#
+# Copyright (c) 2020 Airbyte
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+
+
+from typing import Dict
+
+import requests
+
+
+class ZuoraAuthenticator:
+    def __init__(self, is_sandbox: bool = True):
+        self.is_sandbox = is_sandbox
+
+    @property
+    def endpoint(self) -> str:
+        if self.is_sandbox:
+            return "https://rest.apisandbox.zuora.com"
+        else:
+            return "https://rest.zuora.com"
```

Comment:
could be shortened in one line: 
`return f"https://rest.{'apisandbox.' if self.is_sandbox else ''}zuora.com"`
or 
`return "https://rest.{}zuora.com".format('apisandbox.' if self.is_sandbox else '')`

Reply:
In the future there could be some more endpoints for the connection, full list [is here](https://www.zuora.com/developer/api-reference/), under the section of `Access to the API`. If so, it. could be hard to read the one-line solution. But I agree, for now, it could be changed to what you're suggesting.

Reply:
OK